frontend/packages/console-dynamic-plugin-sdk/src/api/api.ts

```diff
@@ -8,3 +15,16 @@ const MockImpl = () => {
 
 export const useK8sWatchResource: UseK8sWatchResource = MockImpl;
 export const useK8sWatchResources: UseK8sWatchResources = MockImpl;
+
+// status, icons
+export const Status: React.FC<StatusProps> = MockImpl;
```

Comment:
Maybe we can split this file into `api-hooks`, `api-components`, `api-utils` etc.

Can be done later.